lib/v2/bnu/dwxgb.js

```diff
@@ -25,20 +25,21 @@ module.exports = async (ctx) => {
     const $ = cheerio.load(response.data);
 
     const list = $('ul.container.list > li')
-        .map((_, item) => ({
+    .map((_, item) => {
+        const link = $(item).find('a').attr('href');
+        const absoluteLink = new URL(link, currentUrl).href;
+        return {
             title: $(item).find('a').text().trim(),
             pubDate: parseDate($(item).find('span').text()),
-            link: `${rootUrl}/${category}/${type}/${$(item).find('a').attr('href')}`,
-        }))
-        .get();
+            link: absoluteLink,
+        };
+    })
+    .get();
 
     const items = await Promise.all(
         list.map((item) =>
             ctx.cache.tryGet(item.link, async () => {
-                const detailResponse = await got({
-                    method: 'get',
-                    url: item.link,
-                });
+                const detailResponse = await got(item.link);
```

Comment:
Since you've changed this, why not change 
https://github.com/DIYgod/RSSHub/blob/5b1c44f2c37542ed920dcc155ab7b1cc46621635/lib/v2/bnu/dwxgb.js#L12-L23
to the same style as well.